tests/api.spec.js

```diff
@@ -2,10 +2,10 @@
 const axios = require('axios');
 require('dotenv').config();
 const bcrypt = require('bcrypt');
-const jwt = require('jsonwebtoken');
+// const jwt = require('jsonwebtoken');
 const { SERVER_ADDRESS = 'http://localhost:', PORT = 3000 } = process.env;
 const API_URL = process.env.API_URL || SERVER_ADDRESS + PORT;
-const { JWT_SECRET = 'neverchane' } = process.env;
+// const { JWT_SECRET = 'neverchane' } = process.env;
```

Comment:
It is okay to import JWT_SECRET into this file, but its value should be stored in the .env file.